app/controllers/content_items_controller.rb

```diff
@@ -1,6 +1,5 @@
 class ContentItemsController < ApplicationController
   include Slimmer::Template
```

Comment:
If this is a revert of https://github.com/alphagov/government-frontend/pull/2156/, this line can be removed:

```suggestion
```

Reply:
👍